src/components/GameDetail/GameProgress.vue

```diff
@@ -0,0 +1,423 @@
+<template lang="html">
+  <div :class="['game-progresses', { small }]">
+    <div v-if="hasProgress && !showProgressField" class="progress">
+      <div
+        class="progress-data"
+        v-if="!pie"
+      >
+        <progress
+          class="progress-bar"
+          max="100"
+          :value="localProgress.number"
+        >
+          <div class="progress-bar-fallback">
+            <span :style="style">
+              Progress: {{localProgress.number}}%
+            </span>
+          </div>
+        </progress>
+        <div
+          v-if="localProgress.number"
+          class="progress-bar-label"
+        >
+          {{localProgress.number}}%
+        </div>
+      </div>
+
+      <div
+        v-else
+        class="progress-pie"
+        :data-value="localProgress.number"
+      >
```

Comment:
consider inverting these two blocks so you don't have to use the NOT operator:

```
      <div
        v-if="pie"
        class="progress-pie"
        :data-value="localProgress.number"
      >

      ...

      <div
        class="progress-data"
        v-else
      >
```

might be easier to read IMO